internal/jennies/golang/rawtypes.go

```diff
@@ -109,7 +109,11 @@ func (jenny RawTypes) generateSchema(context languages.Context, schema *ast.Sche
 
 		customMethodsBlock := template.CustomObjectMethodsBlock(object)
 		if jenny.tmpl.Exists(customMethodsBlock) {
-			innerErr = jenny.tmpl.RenderInBuffer(&buffer, customMethodsBlock, map[string]any{
+			innerErr = jenny.tmpl.Funcs(map[string]any{
+				"formatPkg": func(pkg string) string {
```

Comment:
Why not sticking with existing helper names?
See the builder.go jenny: https://github.com/grafana/cog/blob/08b35448aa1db9214d9e949a3f1093a7d5104767/internal/jennies/golang/builder.go#L102-L104